load.php

```diff
@@ -248,6 +254,39 @@ function perflab_can_load_module( $module ) {
 	return (bool) $module();
 }
 
+/**
+ * Checks whether the given module has already been loaded by a separate plugin.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $module Slug of the module.
+ * @return bool Whether the module has already been loaded by a separate plugin.
+ */
+function perflab_is_standalone_plugin_loaded( $module ) {
+	$standalone_plugins_constants = perflab_get_standalone_plugins_constants();
+	if (
+		isset( $standalone_plugins_constants[ $module ] ) &&
+		defined( $standalone_plugins_constants[ $module ] ) &&
+		! str_starts_with( constant( $standalone_plugins_constants[ $module ] ), 'Performance Lab ' )
```

Comment:
Using the content of the constant is fine for now, but it may be good to iterate on this approach so we are not dependent on modifying the constant value during the build process when a module is converted to a standalone plugin.

Reply:
What is the drawback of converting it as part of the build step?

Reply:
The only real drawback is that it couples this execution logic with our build process, which means that changes to our build process could introduce bugs. If we could avoid this coupling, I think it would be nice. Not a blocker.

Reply:
Thanks @joemcgill, Do we need to open a follow-up issue for this? If yes, can you do it, or would you like me to open it?

Reply:
I see that you already have opened an issue for this, which I think is the right next step. Thank you!